modules/heat_conduction/doc/content/source/actions/CoupledHeatTransferAction.md

```diff
@@ -0,0 +1,6 @@
+# Coupled Heat Transfer Action
+
+## Description
+
+The `CoupledHeatTransferAction` action sets up all of the necessary objects for
+doing transfer between heat conduction and thermal hydraulics module.
```

Comment:
Mention that this action couples a 1D flow channel in THM to 3D heat conduction.

Reply:
Change the class doxygen and `addClassDescription` accordingly.

Reply:
It also works with 2D heat conduction.